Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1349,3 +1354,63 @@ private class BrowserScreenshotHelper: ScreenshotHelper {
         return nil
     }
 }
+
+private class BrowserPrintPageRenderer: UIPrintPageRenderer {
+    private weak var browser: Browser?
+    let textAttributes = [NSFontAttributeName: UIFont(name: "Helvetica", size: 8)!]
+    let dateString: String
+
+    init(browser: Browser) {
+        self.browser = browser
+        let dateFormatter = NSDateFormatter()
+        dateFormatter.dateStyle = .ShortStyle
+        dateFormatter.timeStyle = .ShortStyle
+        self.dateString = dateFormatter.stringFromDate(NSDate())
+
+        super.init()
+
+        self.footerHeight = 0.5 * 36
```

Comment:
With constants like this, we've been using UX-structs at the top of the file. If you move this to its own file, something like:

struct BrowserPrinterUX {
    let PageInsets = 36
}